apache/superset
# chore: Removes hard-coded colors from the plugins - iteration 1

### SUMMARY
- Replaces hard-coded colors with colors from the theme
- Fixes some storybook properties that were preventing some plugins to render
- Fixes the theme decorator to correctly pass the theme object

### TESTING INSTRUCTIONS
- Open the plugins' storybook
- Check the changed plugins

### ADDITIONAL INFORMATION
